### ISSUE TITLE
on homolog tab on gene page show where there is phenotype data

### ISSUE THREAD

**Issue by mellybelly** [monarch-initiative/monarch-legacy#567]
It would be great to know before you click on a gene which homolog has phenotype data

@harryhoch @nlwashington wouldn't that be awesome?

**Comment by nlwashington** [comment]
yep... this will require some precomputation to make it a fast lookup.  see #380.

**Comment by harryhoch** [comment]
agreed. @kshefchek, did you ever run any SciGraph tests as suggested in https://github.com/monarch-initiative/monarch-app/issues/380?

**Comment by kshefchek** [comment]
I don't believe scigraph currently stores gene phenotype associations, @cmungall @ccondit correct me if I'm wrong.  So we could go the precompute route, generate these via iterative NIF calls, or wait for this to be in SciGraph.  I'm happy to work on it once we decide.

**Comment by nlwashington** [comment]
I believe both orthology relationships as well as gene-phenotypes are on the development instance (geoffery) scigraph (instead of kato).

@ccondit may need to help write some cypher queries to answer this question.  FYI, the orthologous relationships between the genes ought to be subclasses of RO:0002162.

this would be a great thing to add to the agenda for the project meeting for during a breakout!

**Comment by ccondit** [comment]
If the information is in the ontology then it should also be in SciGraph. Is it a matter of filtering the `neighborhood` resource by edge type?

**Comment by nlwashington** [comment]
We need a scigraph call (and/or stored in golr) that includes the phenotypes annotated to orthologs. The request here would be q=gene_id, then we find it's orthologs, and then have the list of phenotypes for each of them.  At first, it could just return a yes or no flag if the ortholog (in a given species) has phenotypes.  But eventually we should be returning the actual phenotypes so we can facet on them.  @ccondit we should add this as a new precomputed service related to https://github.com/SciCrunch/SciGraph/issues/66
